ruby/vampires.rb

```diff
@@ -0,0 +1,137 @@
+# This is a semi-working product with comments at the end of where I had trouble.
+
+question_cycles = 0
+vampire_verdict = nil
+# Ask how many employees are being processed, convert to integer
+puts "How many employees are being processed?"
+num_of_employees = gets.chomp.to_i
+
+until question_cycles == num_of_employees
+
+	# Gather Employee Data
+
+	# Ask employee for name
+	puts "What is your name?"
+	employee_name = gets.chomp.downcase
+
+	# Ask employee how old they are and convert to an integer
+	puts "How old are you?"
+	employee_age = gets.chomp.to_i
+
+	# Ask employee what year they were born and convert to an integer
+	puts "What year were you born?"
+	birth_year = gets.chomp.to_i
+
+	# Ask employee if they will want garlic bread 
+	puts "Our company cafeteria servers garlic bread. Should we order some for you? (y or n)"
+	garlic_bread = gets.chomp.downcase
+
+	# Ask employee if they will want to enroll in company health insurance
+	puts "Would you like to enroll in the company's health insurance? (y or n)"
+	health_insurance = gets.chomp.downcase
+
+
+	# Did the employee get their age right?
+
+	# Get the current year so that we can subtract from it
+	# Create a variable for the current year
+	current_year = Time.now.year
+
+	# Subtract the year they gave from the current year to get how old they should be and store
+	# that in a new variable
+	age_check = current_year - birth_year
+
+	# Match the age_check to the employee_age to see if they match up
+		if employee_age == age_check
+			correct_age = true
+		elsif employee_age == age_check - 1
```

Comment:
I didn't want to just put 2016 as the year to test from so I used Time.now.year. However, I couldn't get it down to the day since someone born in 2000 can be either 15 or 16 right now. So the age_check - 1 can account for someone who was born in 2000 but whose birthday has yet to pass this year.